rust/gitxetcore/Cargo.toml

```diff
@@ -111,12 +111,14 @@ prometheus = "0.13.0"
 prometheus_dict_encoder = { path = "../prometheus_dict_encoder" }
 utime = "0.3.1"
 
-
 # Need to specify this as optional to allow the openssl/vendored option below
 openssl = { version = "0.10", features = [], optional = true }
 
 [target.'cfg(not(windows))'.dependencies]
-openssl = "0.10" 
+openssl = { version = "0.10", features = ["vendored"]}
```

Comment:
Don't force vendoring of OpenSSL, this will not go over well with downstream Linux distros that try to package this for you.